source/MRMesh/MRObjectMeshHolder.h

```diff
@@ -119,9 +119,14 @@ class MRMESH_CLASS ObjectMeshHolder : public VisualObject
     ObjectMeshHolder( ProtectedStruct, const ObjectMeshHolder& obj ) : ObjectMeshHolder( obj )
     {}
 
-    const MeshTexture& getTexture() const { return texture_; }
-    virtual void setTexture( MeshTexture texture ) { texture_ = std::move( texture ); dirty_ |= DIRTY_TEXTURE; }
-    virtual void updateTexture( MeshTexture& updated ) { std::swap( texture_, updated ); dirty_ |= DIRTY_TEXTURE; }
+    // for backward compatibility
+    MRMESH_API const MeshTexture& getTexture() const;
```

Comment:
please move it away from `// for backward compatibility` and write detailed comment